.evergreen/csfle/README.md

```diff
@@ -0,0 +1,76 @@
+# Scripts for In-Use Encryption testing
+
+This folder contains scripts for use with In-Use Encryption.
+
+See the [Azure KMS](./azurekms/README.md) and [GCP KMS](./gcpkms/README.md)
+for more information on those specific scenarios.
+
+## Prerequisities
+
+The system you are running on must have Python 3 and have access to the
+`drivers/csfle` [AWS Vault](https://wiki.corp.mongodb.com/display/DRIVERS/Using+AWS+Secrets+Manager+to+Store+Testing+Secrets).
+For legacy usage, see below.
+
+## Usage
+
+Set up In-Use Encryption by first fetching the secrets and then launching the kms servers:
+
+```bash
+$DRIVERS_TOOLS/.evergreen/csfle/setup_secrets.sh
+$DRIVERS_TOOLS/.evergreen/csfle/start_servers.sh
+```
+
+The generated `secrets-export.sh` file can be sourced from your `cwd` to get the required credentials for testing.
+
+The following servers will be started:
+
+- Mock KMIP server on port 5698
+- KMS HTTP server with an expired cert on port 8000
+- KMS HTTP server with an "wrong host" cert on port 8001
+- KMS HTTP server with a correct cert on port 8002
+- Mock Azure IMDS server on port 8080
+
+When finished, stop the servers by running:
+
+```bash
+$DRIVERS_TOOLS/.evergreen/csfle/stop_servers.sh
+```
+
+It is recommended that you start the servers in the background, and then use the `await_servers.sh` script.
```

Comment:
What does `await_servers.sh` do? The instructions above suggest that I run `setup_secrets.sh` and then `start_servers.sh`, do I run `start_servers.sh` in the background?

Reply:
In the Python driver, we can start it in the foreground because it is part of the same task.   In the Go example, which is probably more common, the servers are started in a separate task, and `await_servers.sh` is used to wait for them before running the test scripts.

Reply:
I clarified in the readme.